tree/30_generic_methods/package_install_version_cmp.cf

```diff
@@ -36,8 +36,11 @@ bundle agent package_install_version_cmp(package_name, version_comparator, packa
 {
   vars:
       "class_prefix"     string => canonify(join("_", "this.callers_promisers"));
+      "args"              slist => { "${package_name}", "${version_comparator}", "${package_version}", "${action}" };
 
   methods:
       "install or update"  usebundle => package_install_version_cmp_update("${package_name}", "${version_comparator}", "${package_version}", "${action}", "true");
       "new result classes" usebundle => _classes_copy("${class_prefix}_install_or_update", "${class_prefix}");
+
+      "report"             usebundle => _logger("Install package ${package_name} version ${package_version}", "", @{args});
```

Comment:
if all level are reporting, how can we prevent from having duplicate lines in reports ?